### DISCUSSION TITLE
How to use prebuilt kernel Image to flash

### DISCUSSION THREAD

**Discussion by CHNtentes** [OE4T/meta-tegra#1203] [Q&A]
Hi. I build a kernel `Image` file elsewhere, and I want to use it instead of kernel built in yocto.

I know I can simply use `abootimg` to create `boot.img`, then replace the one in tegraflash tar.gz. 

But I wonder if I can do this during build? If so, which recipe should I modify?

Thanks in advance.

**Comment by madisongh** [comment]
You should probably create a new recipe to fetch your pre-built kernel and kernel modules and install/package them as required, then set `PREFERRED_PROVIDER_virtual/kernel` in your machine configuration to point to the new recipe.
